### PR TITLE
add travis config

### PR DESCRIPTION
**Pull request by paulirish**
we'll see if this config needs some tweaking first..

### CODE REVIEW

**Comment by paulirish** [comment]
image


@denar90 well... shall we require node >= 6?

**Comment by denar90** [comment]
I like using node >= 6.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: node_js
+matrix:
+  include:
+    - node_js: "4.4"
+    - node_js: "5"
+    - node_js: "6"
+    - node_js: "7"
+sudo: required
+dist: trusty
+cache:
```
Comment (denar90) [inline .travis.yml]: What do you think about `cache: yarn` ?
https://docs.travis-ci.com/user/caching/#yarn-cache

Comment (paulirish) [inline .travis.yml]: sg

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  include:
+    - node_js: "6"
+    - node_js: "7"
+sudo: required
+dist: trusty
+cache:
+   directories:
+     - node_modules
+install:
+  - npm install
```
Comment (denar90) [inline .travis.yml]: Can `yarn install` fix https://travis-ci.org/paulirish/pwmetrics/jobs/191284973#L148 ?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+matrix:
+  include:
+    - node_js: "6"
+    - node_js: "7"
+dist: trusty
+cache:
+  yarn: true
+  directories:
+    - node_modules
+install:
```
Comment (denar90) [inline .travis.yml]: What if add ?
```
before_install:
  - npm install -g yarn --cache-min 999999999
```

Comment (denar90) [inline .travis.yml]: Looks like it works with this solution 
https://github.com/paulirish/pwmetrics/pull/29

**Pull request merged**